region/maridia/inner-pink.json

```diff
@@ -1956,8 +1956,28 @@
                     "canSuitlessMaridia",
                     "canDraygonGrappleJump",
                     {"or":[
-                      "Missile",
-                      "Super"
+                      {"and": [
+                        "Missile",
+                        {"cost":[
+                          {"ammo": [
+                            {
+                              "type": "Missile",
+                              "count": 9
```

Comment:
The Grapple Jump requires only one exposed turret, so 3 missiles or 1 super

Reply:
Was under the impression everyone always killed all 3. I don't even know how the Draygon fight plays out with them intact honestly.

Reply:
It's not really about the Draygon fight. The turrets respawn (but don't fire) if you leave the room.